grid/recruitment/get-cv/maps/mock.suma

```diff
@@ -0,0 +1,13 @@
IChtYWNPUyBWZXJzaW9uIDEyLjIuMSBcKEJ1aWxkIDIxRDYyXCkgUXVhcnR6IFBERkNvbnRleHQpCi9DcmVhdG9yIChQYWdlcykgL0NyZWF0aW9uRGF0ZSAoRDoyMDIyMTIwNTE0MTIxNlowMCcwMCcpIC9Nb2REYXRlIChEOjIwMjIxMjA1MTQxMjE2WjAwJzAwJykKPj4KZW5kb2JqCnhyZWYKMCA5NgowMDAwMDAwMDAwIDY1NTM1IGYgCjAwMDAwMDUyNTUgMDAwMDAgbiAKMDAwMDAxMzUzMyAwMDAwMCBuIAowMDAwMDAwMDIyIDAwMDAwIG4gCjAwMDAwMDUzNTkgMDAwMDAgbiAKMDAwMDAwODIwMyAwMDAwMCBuIAowMDAwMDE0NTU2IDAwMDAwIG4gCjAwMDAwMDAwMDAgMDAwMDAgbiAKMDAwMDAyMjI0NSAwMDAwMCBuIAowMDAwMDQ0ODg0IDAwMDAwIG4gCjAwMDAwMDAwMDAgMDAwMDAgbiAKMDAwMDA0NzU2NiAwMDAwMCBuIAowMDAwMDEzNzQyIDAwMDAwIG4gCjAwMDAwMDU0OTAgMDAwMDAgbiAKMDAwMDAwODQwMiAwMDAwMCBuIAowMDAwMDA4MjM5IDAwMDAwIG4gCjAwMDAwMDg1MDkgMDAwMDAgbiAKMDAwMDAxMzc4MiAwMDAwMCBuIAowMDAwMDA4NjMyIDAwMDAwIG4gCjAwMDAwMDg1NzggMDAwMDAgbiAKMDAwMDAwODg5OSAwMDAwMCBuIAowMDAwMDA5MDQ0IDAwMDAwIG4gCjAwMDAwMDkxNDQgMDAwMDAgbiAKMDAwMDAwOTIxNiAwMDAwMCBuIAowMDAwMDA5MzE2IDAwMDAwIG4gCjAwMDAwMDkzODggMDAwMDAgbiAKMDAwMDAwOTQ2MCAwMDAwMCBuIAowMDAwMDA5NTMyIDAwMDAwIG4gCjAwMDAwMDk2MDQgMDAwMDAgbiAKMDAwMDAwOTY3NiAwMDAwMCBuIAowMDAwMDA5NzQ5IDAwMDAwIG4gCjAwMDAwMDk4NTAgMDAwMDAgbiAKMDAwMDAwOTkyNCAwMDAwMCBuIAowMDAwMDA5OTk3IDAwMDAwIG4gCjAwMDAwMTAwNzEgMDAwMDAgbiAKMDAwMDAxMTMyNSAwMDAwMCBuIAowMDAwMDExMzk5IDAwMDAwIG4gCjAwMDAwMTE0NzIgMDAwMDAgbiAKMDAwMDAxMTU0NSAwMDAwMCBuIAowMDAwMDEyNzk5IDAwMDAwIG4gCjAwMDAwMTI4NzMgMDAwMDAgbiAKMDAwMDAxMjk0NiAwMDAwMCBuIAowMDAwMDEzMDE5IDAwMDAwIG4gCjAwMDAwMTMwOTMgMDAwMDAgbiAKMDAwMDAxMzE2NiAwMDAwMCBuIAowMDAwMDEzMjM5IDAwMDAwIG4gCjAwMDAwMTMzMTMgMDAwMDAgbiAKMDAwMDAxMzQ1OSAwMDAwMCBuIAowMDAwMDEwMTcwIDAwMDAwIG4gCjAwMDAwMTA0MDEgMDAwMDAgbiAKMDAwMDAxMDYzMiAwMDAwMCBuIAowMDAwMDEwODYzIDAwMDAwIG4gCjAwMDAwMTEwOTQgMDAwMDAgbiAKMDAwMDAxMDI0OSAwMDAwMCBuIAowMDAwMDEwMzI0IDAwMDAwIG4gCjAwMDAwMTA0ODAgMDAwMDAgbiAKMDAwMDAxMDU1NSAwMDAwMCBuIAowMDAwMDEwNzExIDAwMDAwIG4gCjAwMDAwMTA3ODYgMDAwMDAgbiAKMDAwMDAxMDk0MiAwMDAwMCBuIAowMDAwMDExMDE3IDAwMDAwIG4gCjAwMDAwMTExNzMgMDAwMDAgbiAKMDAwMDAxMTI0OCAwMDAwMCBuIAowMDAwMDExNjQ0IDAwMDAwIG4gCjAwMDAwMTE4NzUgMDAwMDAgbiAKMDAwMDAxMjEwNiAwMDAwMCBuIAowMDAwMDEyMzM3IDAwMDAwIG4gCjAwMDAwMTI1NjggMDAwMDAgbiAKMDAwMDAxMTcyMyAwMDAwMCBuIAowMDAwMDExNzk4IDAwMDAwIG4gCjAwMDAwMTE5NTQgMDAwMDAgbiAKMDAwMDAxMjAyOSAwMDAwMCBuIAowMDAwMDEyMTg1IDAwMDAwIG4gCjAwMDAwMTIyNjAgMDAwMDAgbiAKMDAwMDAxMjQxNiAwMDAwMCBuIAowMDAwMDEyNDkxIDAwMDAwIG4gCjAwMDAwMTI2NDcgMDAwMDAgbiAKMDAwMDAxMjcyMiAwMDAwMCBuIAowMDAwMDEzNjIzIDAwMDAwIG4gCjAwMDAwMTQ0NDkgMDAwMDAgbiAKMDAwMDAxMzg0MCAwMDAwMCBuIAowMDAwMDEzOTUzIDAwMDAwIG4gCjAwMDAwMTQwNzUgMDAwMDAgbiAKMDAwMDAxNDE5NyAwMDAwMCBuIAowMDAwMDE0MzIwIDAwMDAwIG4gCjAwMDAwMTQ5ODIgMDAwMDAgbiAKMDAwMDAxNTIyNCAwMDAwMCBuIAowMDAwMDIzMzYzIDAwMDAwIG4gCjAwMDAwMjI2NzEgMDAwMDAgbiAKMDAwMDAyMzYxMiAwMDAwMCBuIAowMDAwMDQ1MDU3IDAwMDAwIG4gCjAwMDAwNDUyOTIgMDAwMDAgbiAKMDAwMDA0ODI4MyAwMDAwMCBuIAowMDAwMDQ3ODM0IDAwMDAwIG4gCjAwMDAwNDg1MzcgMDAwMDAgbiAKMDAwMDA1ODM4NyAwMDAwMCBuIAp0cmFpbGVyCjw8IC9TaXplIDk2IC9Sb290IDc4IDAgUiAvSW5mbyA5NSAwIFIgL0lEIFsgPDU4ZjQwOTU3MmY2MDg5YTg5NTQ4NzE5YjYwNzZjOWMzPgo8NThmNDA5NTcyZjYwODlhODk1NDg3MTliNjA3NmM5YzM+IF0gPj4Kc3RhcnR4cmVmCjU4NjAzCiUlRU9GCg==','base64'),
```

Comment:
Wow :sweat_smile: I suppose there is no way right now to make this nicer, right?

Reply:
Or actually, this is plain JavaScript, so why not load from a local file - convenience?

Reply:
I wanted to return real resume file from mock provider map.

Reply:
> Or actually, this is plain JavaScript, so why not load from a local file - convenience?

The map is being executed by OneSDK. What local files you mean? There is just profile, map and provider file distributed by registry.

Reply:
Oh no, it's *not* plain javascript, it's a mock map - I thought for a moment that's a test (which could use `fs`, etc.). Brain glitch :)